ui/src/app/utils/leo-runtime-initializer.tsx

```diff
@@ -76,6 +76,20 @@ export const throwRuntimeNotFound = (
   gcePersistentDisk: Disk
 ) => {
   const defaultRuntime = cond<Runtime>(
+    [
+      !currentRuntime && !!gcePersistentDisk,
+      () => {
+        const defaultRuntimeTemplate =
+          runtimePresets().generalAnalysis.runtimeTemplate;
```

Comment:
something to note is that this assumes that we want *this* template as the default (not Hail).

I think that is correct, and it is what we do when there's no PD.  Just something to consider.

Reply:
A comment like the one on L93 would be helpful here

Reply:
Good point. I left a comment.